mx.truffleruby/mx_truffleruby.py

```diff
@@ -68,3 +112,564 @@ def ruby_testdownstream(args):
     'deploy-binary-if-truffle-head': [deploy_binary_if_truffle_head, ''],
     'ruby_testdownstream': [ruby_testdownstream, '']
 })
+
+if 'RUBY_BENCHMARKS' in os.environ:
```

Comment:
Could we have all this stuff in a different python file like `mx_truffleruby_benchmarks.py` and only load it when needed?
It's a lot of code and I would rather isolate it better from the build.